itest/rust/src/register_tests/icon_test.rs

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) godot-rust; Bromeon and contributors.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+use godot::prelude::*;
+
+use crate::framework::itest;
+
+const REF_COUNTED_ICON: &str = "res://icons/ref_counted_icon.svg";
```

Comment:
Aren't paths to all these icons invalid? Don't we check the one and the same case?